sensor/common/registry/registry_store.go

```diff
@@ -55,6 +57,12 @@ type Store struct {
 
 	// centralRegistryIntegration holds registry integrations sync'd from Central.
 	centralRegistryIntegrations registries.Set
+
+	// holds the mirror sets used for managing registry mirrors.
+	icspRules      map[string]*operatorV1Alpha1.ImageContentSourcePolicy
+	idmsRules      map[string]*configV1.ImageDigestMirrorSet
+	itmsRules      map[string]*configV1.ImageTagMirrorSet
```

Comment:
I'm not sure how we envision this to be called, but we could also do `map[types.UID]...` instead. Any reason not to do that? Seems like this is more natural than calling e.g. `DeleteIMageConentSourcePolicy(string(types.UID))`.

Reply:
No reason why not to do that, I'll change it to use `types.UID` - other usages of the UIDs in ACS are converted to string - thought was to be consistent - but the conversions were needlessly verbose, this is cleaner. Thanks!